[{"ContainerToolsException.Message":"Docker command failed with exit code 1. #0 building with \"default\" instance using docker driver  #1 [internal] load .dockerignore #1 transferring context: 2B done #1 DONE 0.0s  #2 [internal] load build definition from Dockerfile #2 transferring dockerfile: 31B done #2 DONE 0.0s ERROR: failed to solve: the Dockerfile cannot be empty","ContainerToolsException.ErrorCode":"CTC1014","ContainerToolsException.StackTrace":"   at Microsoft.VisualStudio.Containers.Tools.Common.Client.DockerClient.<BuildImageAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<RebuildImageAsync>d__11.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchForDockerfileAsync>d__7.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Shared.ContainerLaunchManager.<BuildAndLaunchAsync>d__5.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildAndLaunch.<ExecuteCoreAsync>d__200.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.VisualStudio.Containers.Tools.Tasks.ContainerBuildTask.<ExecuteAsync>d__30.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.FaultingUserFile.FilePath":"Dockerfile","ContainerToolsException.FaultingUserFile.LineStart":"","ContainerToolsException.FaultingUserFile.ColumnStart":"","ContainerToolsException.FaultingUserFile.LineEnd":"","ContainerToolsException.FaultingUserFile.ColumnEnd":"","ContainerToolsException.CommandLine.Program":"docker","ContainerToolsException.CommandLine.TelemetrySafeArguments":"build","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":"#0 building with \"default\" instance using docker driver\r\n\r\n#1 [internal] load .dockerignore\r\n#1 transferring context: 2B done\r\n#1 DONE 0.0s\r\n\r\n#2 [internal] load build definition from Dockerfile\r\n#2 transferring dockerfile: 31B done\r\n#2 DONE 0.0s\r\nERROR: failed to solve: the Dockerfile cannot be empty"}]